integration/test_named_vectors.py

```diff
@@ -756,3 +756,39 @@ def test_deprecated_syntax(collection_factory: CollectionFactory):
             return_metadata=wvc.query.MetadataQuery.full(),
         )
     assert "Providing lists of lists has been deprecated" in str(e)
+
+
+def test_fetch_objects_on_reference(client_factory: ClientFactory) -> None:
```

Comment:
Just a couple of things on this one:

- Can it be renamed to `test_include_vector_on_references`?
- Can you parametrise the test like we do [here](https://github.com/weaviate/weaviate-python-client/blob/main/integration/test_named_vectors.py#L44) so that all three cases of `False, ["bringYourOwn1"], True` are tested?
 
You can have `include_vector,expected` as the parametrised variables and map `expected` to the value of `objs[0].references["hasRef"].objects[0].vector` in an assertion

Reply:
No, haha it's alright. It's updated, however, I added the True test but I commented it as the test won't pass until https://github.com/weaviate/weaviate/issues/6279 is resolved.